src/Umbraco.Web.UI.Client/src/views/components/content/umb-content-node-info.html

```diff
@@ -26,7 +26,7 @@
             <umb-box-header title-key="redirectUrls_redirectUrlManagement"></umb-box-header>
             <umb-box-content class="block-form">
                 <div style="position: relative;">
-                    <div ng-if="loadingRedirectUrls" style="background: rgba(255, 255, 255, 0.8); position: absolute; top: 0; left: 0; right: 0; bottom: 0;"></div>
+                    <div ng-if="loadingRedirectUrls" style="background: rgba(255: 255, 255, 0.8); position: absolute; top: 0; left: 0; right: 0; bottom: 0;"></div>
```

Comment:
Is the colon replacing the comma for a reason? (not sure if this is something new I am unaware of)

Reply:
Nope, good catch 👍